core/controllers/community_dashboard_test.py

```diff
@@ -120,7 +122,7 @@ def setUp(self):
             'topic_name': 'topic',
             'story_title': 'title 0',
             'chapter_title': 'Node1',
-            'content_count': 2,
+            'content_count': 3,
```

Comment:
This and subsequent increments is because the modified exploration now has an 'EndExploration' state as well (since it has to be published) and so the content in that has to be taken into account.

Reply:
Could you please add this as a comment in the code? (If you have to leave a note for reviewers, it's probably also worth leaving it for future developers.)

Reply:
Done